packaging/bin/create-msi-repo.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+set -eEuo pipefail
+
+# External dependencies:
+# - https://github.com/s3tools/s3cmd
+#   s3cmd expects AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY to be set in the
+#   environment.
+# - generate_index.py
+#   For generating the index.html of each directory. It's available in the
+#   packaging/bin directory of the k6 repo, and should be in $PATH.
+
+_s3bucket="${S3_BUCKET-dl.k6.io}"
+_usage="Usage: $0 <pkgdir> <repodir> [s3bucket=${_s3bucket}]"
+PKGDIR="${1?${_usage}}"  # The directory where .msi files are located
+REPODIR="${2?${_usage}}" # The package repository working directory
+S3PATH="${3-${_s3bucket}}/msi"
+# Number of packages to keep, older packages will be deleted.
+RETAIN_PKG_COUNT=25
+
+log() {
+    echo "$(date -Iseconds) $*"
+}
+
+delete_old_pkgs() {
+  find "$1" -name '*.msi' -type f | sort -r | tail -n "+$((RETAIN_PKG_COUNT+1))" | xargs -r rm -v
+}
```

Comment:
Sure, we could drop it for now. My main motivation was that if we don't have some cleanup in place the time and costs will keep increasing for every release. s3cmd does preserve `mtime` and we could filter by that. But OK, I'll remove this for now and we can add cleanup later.

Reply:
I am fine with keeping it, just with `mtime` instead of just a count ... But also I don't think it matters that much so :shrug: 

WDYT @na-- , you wanted to keep packages forever

Reply:
I don't have strong opinion either way. The data hoarder in me balks at deleting anything, but my main problem with keeping them around forever is that it would increase release times, not so much the costs, so I understand why we have to delete them. And I see you already use `mtime`, which I somewhat prefer, so :+1: for resolving this?